src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -182,7 +182,9 @@ public int getTotalUserCount(){
 	          String text = (String) entity.getProperty("text");
 	          long timestamp = (long) entity.getProperty("timestamp");
 
-	          Message message = new Message(id, sender, text, timestamp, recipient);
+            Message message = new Message(id, sender, text, timestamp, recipient);
```

Comment:
just for ease of reading the code, i would fix the indentation to tab over one more (to be within the try block)

Reply:
I did not catch this, thanks